outplacement/__manifest__.py

```diff
@@ -1,6 +1,6 @@
 {
     'name': 'Outplacement',
-    'version': '12.0.0.1',
+    'version': '12.0.1.1.0',
```

Comment:
Version number should either be 12.0.1.0.0 or 12.0.0.2.0 in this case, I believe the later or is backward incompatibility introduced in this version?
Also 1.0.0 is generally reserved for first production version so its probably 12.0.0.2.0 you want.